src/Services/src/File/FileService.cs

```diff
@@ -13,6 +13,8 @@ public class FileService : IFileService
         private const string _defaultLogsDir = "Logs";
         private const string _defaultCfCliDir = ".cf";
         private const string _defaultCfCliConfigFileName = "config.json";
+        private const string _cfDebugAdapterScriptDir = "Resources";
+        private const string _cfDebugAdapterScriptName = "CFSsh.exe";
```

Comment:
There may be a better way to derive exe name, such as using `nameof`.

Also, an exe is not a script so a more apt var name is needed.

Reply:
> Where are the binaries CFSsh.{dll,exe} coming from? Binaries should not be commited to source repos.

These are rebuilt for every change to `CFSsh` & copied over (manually... gross, I know) to the vsix resources folder so they get shipped in the vsix file. 
We bundle `cf7.exe` & `cf6.exe` similarly -- is that an issue? What's an alternative way to make sure these binaries are available when someone goes to install & use this extension? `<IncludeInVSIX>true</IncludeInVSIX>` is the only way I've seen so far, but happy to look into other options.

Reply:
re cf{6,7}.exe, opened this: #371